serve/mlc_serve/engine/staging_engine_worker.py

```diff
@@ -114,8 +112,13 @@ def add(self, request_states: list[RequestState]):
                 if (
                     request_state.validation_err is not None
                     or request_state.prompt_len >= self.max_context_length
+                    or request_state.prompt_len >= self.max_num_batched_tokens > 0
+                    or (self.cache_manager.get_kv_cache_size() - request_state.prompt_len) < 0
```

Comment:
Why do we need this? At this point we know that `prompt_len < max_num_batched_tokens`. So we can process this request, after evicting the cache if necessary.

Reply:
assuming that PR #85 verifies this and get_kv_cache_size > max_num_batched_tokens your comment is correct, we can remove this condition

Reply:
fixed